src/vec_buffers.h

```diff
@@ -101,23 +101,23 @@ class Buffers {
     Buffers(const Buffers<T>& vec, int n = 0);
     Buffers(const Buffers<T>& vec, int begin, int end);
     Buffers(const Buffers<T>& vec1, const Buffers<T>& vec2);
-    virtual ~Buffers();
-    virtual int get_n(void) const;
-    virtual size_t get_size(void) const;
+    ~Buffers();
```

Comment:
This is safe because nobody inherits from this class anymore, but if someone want to inherit from it it must remember to make the destructor virtual again and this can be easily overlooked.

To be extra-safe, you should declare the class as `class Buffers final {` at l.97

That way, you cannot inherit from `Buffers` (and if you really want to, you need to come edit the class to remove the `final` and if you do that, you are more likely to also edit the destructor to make it virtual again).